assemble/bin/accumulo

```diff
@@ -72,13 +72,6 @@ function main() {
   export HADOOP_HOME=$HADOOP_PREFIX
   export HADOOP_HOME_WARN_SUPPRESS=true
 
-  if [ -f "${conf}/jaas.conf" ]; then
-    export JAVA_OPTS=("${JAVA_OPTS[@]}" "-Djava.security.auth.login.config=${conf}/jaas.conf")
```

Comment:
We don't need the JAAS configuration right now for ourselves, but that's only because of our use of DIGEST ACLs for authentication with ZooKeeper. I think I have an issue somewhere to switch to using SASL ACLs when we're using Kerberos elsewhere (but I can't find it at the moment).

It _should_ be implied if that there is some code that needs to be deployed inside Accumulo that needs to perform a login via JAAS, that the user can specify their own JAAS configuration file. I don't think we need to be explicit (for now).